### DISCUSSION TITLE
MONAILabel module should allow user to select different volumes in current scene

### DISCUSSION THREAD

**Discussion by xmszj** [Project-MONAI/MONAILabel#1354] [Ideas]
this selector bar is gray.

![image](https://user-images.githubusercontent.com/85673587/227057645-8809cdc8-4dbe-4c8d-8604-b69438c26b06.png)

**Comment by SachidanandAlle** [comment]
Because there are no labels submitted yet.. and available for training

**Reply by xmszj** [reply]
If I load many nii.gz volumes into the current scene,I want label every volume then submit them ,then how to do?
the button next sample is to load a new volume from the datasets,but it doens't support loading data from the current scene

**Reply by SachidanandAlle** [reply]
You can copy labels into dataset/labels/final folder..

**Reply by diazandr3s** [reply]
Here is a similar discussion with more questions/comments: https://discourse.slicer.org/t/monailabel-module-should-allow-user-to-select-different-volumes-in-current-scene/28522/14
